demux/cli/samplesheet.py

```diff
@@ -138,6 +141,62 @@ def get_project(project):
         # add [section] header
         click.echo('[Data]')
 
+    if application == 'nova':
+        if dualindex:
+            click.echo(click.style(f"No need to specify dual or single index for NovaSeq sample "
+                                   f"sheets, please use --shortest, --longest, or --indexlength "
+                                   f"only!", fg='red'))
+            context.abort()
+
+        lims_keys = ['fcid', 'lane', 'sample_id', 'sample_ref', 'index', 'index2', 'sample_name',
+                     'control', 'recipe', 'operator', 'project']
+        header = [Samplesheet.header_map[head] for head in lims_keys]
+
+        path = os.path.abspath(__file__)
+        dir_path = os.path.dirname(path)
+        with open(f"{dir_path}/../../files/20181012_Indices.csv") as csv_file:
+            dummy_samples_csv = csv.reader(csv_file, delimiter=',')
+            dummy_samples = [row for row in dummy_samples_csv]
```

Comment:
What does this do?

Reply:
Because I need to iterate over the csv-file once for every lane, I create a list with every row in the csv.reader object. The csv.reader object closes after one iteration, but I can reuse the list in line 159.